### PR TITLE
DEP Remove unsupported modules

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/recipe-kitchen-sink/issues/29

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -9,10 +9,7 @@
         "silverstripe/recipe-cms": "5.x-dev",
         "silverstripe/blog": "4.x-dev",
         "silverstripe/widgets": "3.x-dev",
```
Comment (maxime-rainville) [inline composer.json]:
```suggestion
```

File: composer.json [hunk 2]
```diff
@@ -9,10 +9,7 @@
...
-        "silverstripe/content-widget": "3.x-dev",
         "silverstripe/spamprotection": "4.x-dev",
```
Comment (maxime-rainville) [inline composer.json:11]: Is there any point having spam protection in here if comments are not there.

File: phpunit.xml.dist [hunk 3]
```diff
@@ -5,10 +5,7 @@
 			<directory>vendor/silverstripe/blog/tests</directory>
 			<directory>vendor/silverstripe/lumberjack/tests</directory>
 			<directory>vendor/silverstripe/widgets/tests</directory>
```
Comment (maxime-rainville) [inline phpunit.xml.dist]:
```suggestion
```

**Review by maxime-rainville — CHANGES_REQUESTED** [review]
Did we do a booboo here by  shipping the recipe for CMS 5 with the unsupported module?

File: composer.json (hunk 2 above)
Comment (emteknetnz) [reply composer.json:11]: Probably not, though spam protection is still supported so it should stay there

File: composer.json (hunk 1 above)
Comment (emteknetnz) [reply composer.json]: Looks like we did make a mistake including this in 2.0.0

I've created a changelog PR to mention this - https://github.com/silverstripe/developer-docs/pull/283

**Review by maxime-rainville — APPROVED** [review]

**Pull request merged**